lib/checkPullRequestReview.js

```diff
@@ -287,6 +288,75 @@ const handlePullRequestReview = async (context) => {
   }
 };
 
+/**
+ * This function checks when a pull request gets commented on by the PR author.
+ * If pr author comments on the PR asking reviewers please to take a look (ptal),
+ * We check if the reviewers have been assigned and if not, assign them.
+ *
+ * @param {import('probot').Context} context
+ */
+const handleResponseToReview = async (context) => {
+  /**
+   * @type {import('probot').Octokit.IssuesGetResponse}
+   */
+  const pullRequest = context.payload.issue;
+  const isPullRequest = 'pull_request' in pullRequest;
+  if (!isPullRequest) {
+    return;
+  }
+
+  /**
+   * @type {import('probot').Octokit.IssuesGetCommentResponse}
+   */
+  const comment = context.payload.comment;
+  const commenter = comment.user.login;
+  const commenterIsAuthor = commenter === pullRequest.user.login;
+  if (!commenterIsAuthor) {
+    return;
+  }
+
+  const containsPtalComment = PTAL_COMMENTS.some((item) =>
+    comment.body.toLowerCase().includes(item.toLowerCase())
+  );
+  if (!containsPtalComment) {
+    return;
+  }
+
+  // Pause for 3 minutes in case author wants to perform
+  // the required actions.
+  await utilityModule.sleep(THREE_MINUTES);
```

Comment:
One suggestion regarding the 3 minutes thing - we should also add this to the case when we ask author to assign the reviewers if they are the codeowners - I have noticed that the comment pops really quick even when I am about to assign the reviewer. This one to be specific: https://github.com/oppia/oppiabot/blob/master/lib/checkPullRequestLabels.js#L117. You can update it here or in a separate PR as well.

Reply:
Do you think the 3 minutes wait time is needed here?
The response here should be immediate I believe.

Reply:
See [this](https://github.com/oppia/oppia/pull/10471#issuecomment-682422635) for example, I already assigned a reviewer but oppiabot still leaves a comment. So, we can keep the wait time shorter but we should definitely have it. WDYT?

Reply:
Okay.
I'll add it in a separate PR where I'll also be adding features for the new default changelog label.

Reply:
Sounds good, thanks!